cluster/config-defaults.yaml

```diff
@@ -511,6 +525,18 @@ audittrail_url: ""
 {{end}}
 audittrail_root_account_role: ""
 
+# kube-apiserver / kube-proxy container image name.
+# Temporarly added to help during the Kubernetes service network range migration.
+kube_apiserver_container_image_name: nonexistent.zalan.do/teapot/kube-apiserver:fixed
+kube_proxy_container_image: nonexistent.zalan.do/teapot/kube-proxy:fixed
```

Comment:
I think it's more clear if we make these a true/false flag and template if/else for which image to chose. Otherwise you need to define the correct image in the config-item.